packages/ast_generic/ast_generic.1.8.0/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "Abstract Syntax Tree (AST) supporting 31 programming languages"
+description: """
+This is a library defining a generic AST to factorize similar
+analysis on different programming languages
+(e.g., naming, semantic code highlighting, semgrep matching).
+
+Right now this generic AST is mostly the factorized union of the ASTs of:
+ - Python, Ruby, Lua, Julia, Elixir
+ - Javascript, Typescript, Vue
+ - PHP, Hack
+ - Java, CSharp, Kotlin
+ - C, C++
+ - Go
+ - Swift
+ - OCaml, Scala, Rust
+ - Clojure, Lisp, Scheme
+ - R
+ - Solidity
+ - Bash, Docker
+ - JSON, YAML, HCL, Jsonnet
+
+This is a core library used by Semgrep but which can be of use in other
+projects. This AST is also specified using ATD and so can be leveraged
+from other programming languages such as Typescript, Java, Scala, and
+the other programming languages supported by atdgen.
+"""
+
+maintainer: "Yoann Padioleau <pad@r2c.dev>"
+authors: [ "Yoann Padioleau <pad@r2c.dev>" ]
+license: "LGPL-2.1-only"
+homepage: "https://semgrep.dev"
+dev-repo: "git+https://github.com/returntocorp/semgrep"
+bug-reports: "https://github.com/returntocorp/semgrep/issues"
+
+depends: [
+  "ocaml" {>= "4.12.0"}
+  "dune" {>= "3.2.0" }
+  "commons" {>= "1.8.0"}
+  "lib_parsing" {>= "1.5.5"}
+  "profiling" {>= "1.5.5"}
+  "atdgen" {>= "2.8.0" }
+]
+
+build: ["dune" "build" "-p" name "-j" jobs]
+url {
+  src:
+    "https://github.com/returntocorp/semgrep/releases/download/ast_generic_full_1.8.0/semgrep-full-1.12.0.tar.gz"
```

Comment:
yes, I've just uploaded manually this tgz, but there's no change between 1.8.0 and 1.12.0 regarding ast_generic/. As part of the next release of semgrep we should generate automatically this asset so it will be cleaner.